NOTICE.txt

```diff
@@ -0,0 +1,205 @@
+purescript-react-basic’s codegen is used in this library, under the terms of the
+Apache License 2.0:
```

Comment:
We cannot anything under Apache license in MyCV. CC @mohanzhang

Reply:
That's unusual.  @mohanzhang Can you give more context?